ramls/examples/patron-block-condition.sample

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "3d7c52dc-c732-4223-8bf8-e5917801386f",
+  "name": "Maximum number of items charged out",
+  "blockBorrowings": false,
+  "blockRenewals": false,
+  "blockRequests": false,
+  "valueType": "Integer",
+  "message": "The maximum number of charged out items has been reached"
+}
```

Comment:
Posting this sample fails with "may not be null" "blockBorrowing".

Reply:
fixed